src/main/app/src/app/admin/smart-documents.service.ts

```diff
@@ -62,6 +66,29 @@ export class SmartDocumentsService {
       );
   }
 
+  getTemplatesMappingFlat(
+    zaakafhandelUUID: string,
+  ): Observable<DocumentsTemplateGroup[]> {
+    return this.zacHttp
+      .GET(
+        "/rest/zaakafhandelparameters/{zaakafhandelUUID}/document-templates",
```

Comment:
Maybe not in this already big PR but we could consider moving the `/rest/zaakafhandelparameters` to a constant? I see that in other service classes this is being done. Minor thing really.